// rangoli/index.js


//* Simple Nodejs Code that convert any given String into some Colorfull Text. *//
//* I have Created this module, after I faced so much trouble on debugging on my Node Application, every time Debugging start, I got so confused where the output is and still if i write Some Comment into console, the Output is not much clear and also so frustating.  *//

//* I have created this moudle just to make more convenient the code on Terminal Screen, so that the debugging will be easy and clean, also I have created this module just for my fun purpose also, anyone who thinks can use moudle to write some Fun Code can use it :). Happy Coding  *//


module.exports = {
	prime : function(val){
		return console.log('\x1b[34m', val, '\x1b[0m');
	},
	err : function(val){
		return console.log('\x1b[31m', val, '\x1b[0m');
	},
	succ : function(val){
		return console.log('\x1b[32m', val, '\x1b[0m');
	},
	info : function(val){
		return console.log('\x1b[36m', val, '\x1b[0m');
	},
	warn : function(val){
		return console.log('\x1b[33m', val, '\x1b[0m');
	},
	log : function(val){
		return console.log('\x1b[30m', val, '\x1b[0m');
	}	
}

// 	return{
// 		err : err,
// 		succ: succ,
// 		info: info,
// 		warn: warn,
// 		prim: prim,
// 		log : log,
// 	}
// }
// // var console+"."+log = new Aa(val); 
// var cons = color();
// module.exports(cons);

// cons.prim("hi am primary");
// cons.err("Hi am error msg");
// cons.succ("hi am success msg");
// cons.info("Hi anup how r u?");
// cons.warn("Hello am warning");
// cons.log("Hello am simple");
